ods_ci/tests/Resources/Files/llm/model_expected_responses.json

```diff
@@ -45,6 +45,16 @@
                 }
             }
 
+        },
+        {
+            "query_text": "translate English to German: How old are you?",
+            "models":  {
+                "flan-t5-xl-hf": {
+                    "response_tokens":  6,
+                    "response_text": "Wie alt sind Sie?",
+                    "streamed_response_text":   "{ 'inputTokenCount': 11 } { 'generatedTokenCount': 1, 'text': 'Wie' } { 'generatedTokenCount': 2, 'text': ' alt' } { 'generatedTokenCount': 3, 'text': ' sind' } { 'generatedTokenCount': 4, 'text': ' Sie' } { 'generatedTokenCount': 5, 'text': '?' } { 'generatedTokenCount': 6, 'stopReason': 'EOS_TOKEN' }"
+                }
+            }
```

Comment:
it is ok to have a new query, but what about re-using the one introduced for mt0-xxl? it's always a translation task

Reply:
I though of using it but i think it better to have own queries for different model because all the model will not respond the same answer

Reply:
it doesn't matter, we have entries for each model with their own expected answer